youtube_dl/utils.py

```diff
@@ -91,6 +91,10 @@ def register_socks_protocols():
     'January', 'February', 'March', 'April', 'May', 'June',
     'July', 'August', 'September', 'October', 'November', 'December']
 
+FRENCH_MONTH_NAMES = [
```

Comment:
@phihag first wrote codes for ```ENGLISH_MONTH_NAMES``` in caefb1de877e163fa3ece44757cb2fae6adf47e4. I guess it's because lists are usually used to handle homogeneous data and tuples for heterogeneous ones.